### PR TITLE
[stable/graylog] Modifying README

### PR DESCRIPTION
**Pull request by titansmc**
Only correcting a typo from previous PR @KongZ

### CODE REVIEW

File: stable/graylog/README.md [hunk 1]
```diff
@@ -134,8 +134,8 @@ The following table lists the configurable parameters of the Graylog chart and t
...
-| `graylog.geoip.enabled`                 | If true, Maxmind Geoip Lite will be installed to ${GRAYLOG_HOME}/etc/GeoLite2-City.mmdb                                                               | `false`                               |
-| `graylog.geoip.ddbbUri`                 | If set and geoip enabled,  Maxmind Geoip Lite will be installed from the URL you have defined to ${GRAYLOG_HOME}/etc/GeoLite2-City.mmdb                                                               | ``                               |
+| `graylog.geoip.enabled`                 | If true, Maxmind Geoip Lite will be installed to ${GRAYLOG_HOME}/geoip/GeoLite2-City.mmdb                                                               | `false`                               |
+| `graylog.geoip.mmdbUri`                 | If set and geoip enabled,  Maxmind Geoip Lite will be installed from the URL you have defined to ${GRAYLOG_HOME}/geoip/GeoLite2-City.mmdb                                                               | ``                               |
```
Comment (KongZ) [inline stable/graylog/README.md:138]: Just tiny update, remove double space here `enabled,  Maxmind`

**Comment by KongZ** [comment]
And dont' forget to bump a chart version

**Comment by titansmc** [comment]
@KongZ I am missing something here? Sorry, but I am just getting familiar with PRs

**Comment by KongZ** [comment]
@titansmc one of your commit missing `--signoff` parameter.
Follow this instruction https://github.com/helm/charts/pull/21359/checks?check_run_id=498079703

**Comment by KongZ** [comment]
@titansmc do you still want to open a PR?
You can check what you are missing here
https://github.com/helm/charts/pull/21359/checks

**Pull request closed without merging**